### PR TITLE
intuitionize addcn , subcn , and mulcn

### PR DESCRIPTION
**Pull request by jkingdon**
These are relatively straightforward now that we have #3588 . Most of this follows set.mm fairly closely.

### CODE REVIEW

File: iset.mm [hunk 1]
```diff
@@ -78885,10 +78929,23 @@ sequence rather than the limit of the offsetted sequence (see
...
+     NM, 2-May-1996.) $)
   0ncn $p |- -. (/) e. CC $=
     ( c0 cc wcel cnr cxp 0nelxp df-c eleq2i mtbir ) ABCADDEZCDDFBJAGHI $.
 
+  ${
+    $d A u v x $.
+    $( A complex number is an inhabited set.  Note: do not use this after the
+       real number axioms are developed, since it is a construction-dependent
+       property.  (Contributed by Jim Kingdon, 23-Oct-2023.) $)
+    cnm $p |- ( A e. CC -> E. x x e. A ) $=
```
Comment (tirix) [inline iset.mm:78942]: So should this get a "new usage discouraged" tag?

Comment (tirix) [inline iset.mm:78942]: This is new compared to `set.mm`, but it looks like utlimately you have not used it?

Comment (jkingdon) [reply iset.mm]: Yes. Added.

**Review by tirix — APPROVED** [review]

File: iset.mm (hunk 1 above)
Comment (tirix) [reply iset.mm]: Understood, thanks!

**Pull request merged**